unit_test/test_spdm_responder/digests.c

```diff
@@ -500,6 +500,101 @@ void libspdm_test_responder_digests_case9(void **state)
     assert_int_equal(spdm_context->transcript.message_d.buffer_size, 0);
 }
 
+/**
+ * Test10: a response message is successfully sent ,
+ * Check KeyPairID CertificateInfo and KeyUsageMask
+ * Expected Behavior: requester returns the status LIBSPDM_STATUS_SUCCESS
+ **/
+void libspdm_test_responder_digests_case10(void **state)
+{
+    libspdm_return_t status;
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    size_t response_size;
+    uint8_t response[LIBSPDM_MAX_SPDM_MSG_SIZE];
+    spdm_digest_response_t *spdm_response;
+    uint8_t *digest;
+    spdm_key_pair_id_t *key_pair_id;
+    spdm_certificate_info_t *cert_info;
+    spdm_key_usage_bit_mask_t *key_usage_bit_mask;
+    uint32_t hash_size;
+    uint8_t slot_count;
+    size_t additional_size;
+
+    slot_count = SPDM_MAX_SLOT_COUNT;
+    additional_size = sizeof(spdm_key_pair_id_t) + sizeof(spdm_certificate_info_t) +
+                      sizeof(spdm_key_usage_bit_mask_t);
+    hash_size = libspdm_get_hash_size(m_libspdm_use_hash_algo);
+
+    spdm_test_context = *state;
+    spdm_context = spdm_test_context->spdm_context;
+    spdm_test_context->case_id = 0x0A;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->local_context.capability.flags = 0;
+    spdm_context->last_spdm_request_session_id_valid = false;
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CERT_CAP;
+    spdm_context->connection_info.algorithm.base_hash_algo =
+        m_libspdm_use_hash_algo;
+
+    for (uint8_t index = 0; index < SPDM_MAX_SLOT_COUNT; index++) {
+        spdm_context->local_context.local_cert_chain_provision[index] =
+            &m_libspdm_local_certificate_chain[hash_size *index];
```

Comment:
`hash_size *index` is strange here.
`hash_size` is taken as one certificate_chain size.